addons/flannel/template/base/install.sh

```diff
@@ -33,6 +33,43 @@ function flannel_pre_init() {
     fi
 
     flannel_init_pod_subnet
+
+    if flannel_weave_conflict; then
+        logWarn "The migration from Weave to Flannel will require whole-cluster downtime."
+        logWarn "Would you like to continue?"
+        if ! confirmY ; then
+            bail "Not migrating from Weave to Flannel"
+        fi
+        flannel_check_nodes_connectivity
+    fi
+}
+
+# flannel_check_nodes_connectivity verifies that all nodes in the cluster can reach each other through
+# port 8472/UDP (this communication is a flannel requirement).
+function flannel_check_nodes_connectivity() {
+    local node_count
+    node_count="$(kubectl get nodes --no-headers 2>/dev/null | wc -l)"
+    if [ "$node_count" = "1" ]; then
+        return 0
+    fi
+
+    # if we are in an airgap environment we need to load the kurl-util image locally. this
+    # image has already been loaded in all remote nodes after the common_prompts function.
+    if [ "$AIRGAP" = "1" ] && [ -f shared/kurl-util.tar ]; then
+        if node_is_using_docker ; then
+            docker load < shared/kurl-util.tar
+        else
+            ctr -a "$(${K8S_DISTRO}_get_containerd_sock)" -n=k8s.io images import shared/kurl-util.tar
+        fi
+    fi
+
+    log "Verifying if all nodes can communicate with each other through port 8472/UDP."
+    local kurl_util_image="replicatedhq/kurl-util:$KURL_VERSION"
+    if ! "$DIR"/bin/kurl netutil nodes-connectivity --port 8472 --image "$kurl_util_image" --proto udp; then
```

Comment:
Why not use the KURL_UTIL_IMAGE variable from the Manifest?

```suggestion
    if ! "$DIR"/bin/kurl netutil nodes-connectivity --port 8472 --image "$KURL_UTIL_IMAGE" --proto udp; then
```

![Screen Shot 2023-04-19 at 10 23 20 AM](https://user-images.githubusercontent.com/371319/233152481-455db5ee-e13d-4b4b-8001-5fd284179179.png)

Reply:
Because I am dumb, that is why. Let me adjust that.

Reply:
Fixed.